src/createServer.js

```diff
@@ -1,11 +1,23 @@
-'use strict';
+'use strict'; // eliminates silent errors, works with defaults and more
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const { userRouter } = require('./routes/userRoute');
+const { expenseRouter } = require('./routes/expenseRoute');
+const userService = require('./services/userService');
+const expenseSrvice = require('./services/expensesService');
```

Comment:
There is a typo in the import statement for `expenseService`. It is currently imported as `expenseSrvice`. Correct the spelling to `expenseService` to match the file name and avoid potential errors.